tools/Makefile

```diff
@@ -0,0 +1,51 @@
+CREW_PREFIX = /usr/local
+CREW_LIB_PATH = $(CREW_PREFIX)/lib/crew
```

Comment:
Should this take into account the `/usr/local/lib64` directory on x86_64 architecture?

Reply:
No.  We use /usr/local/lib/crew directory to place crew related stuff even on x86_64 currently.

Reply:
Fair enough.